README.md

```diff
@@ -15,17 +15,19 @@ If you are interested, [check out](https://git.io/Je09Y) my other :octocat: GitH
 
 ___
 
-* [Features](#features)
-* [Prerequisites](#prerequisites)
-* [Usage](#usage)
-  * [Workflow](#workflow)
-  * [Sign commits](#sign-commits)
-* [Customizing](#customizing)
-  * [inputs](#inputs)
-  * [environment variables](#environment-variables)
-* [Keep up-to-date with GitHub Dependabot](#keep-up-to-date-with-github-dependabot)
-* [How can I help?](#how-can-i-help)
-* [License](#license)
+- [About](#about)
+- [Features](#features)
+- [Prerequisites](#prerequisites)
+- [Usage](#usage)
+  - [Workflow](#workflow)
+  - [Sign commits](#sign-commits)
+- [Customizing](#customizing)
+  - [inputs](#inputs)
+  - [outputs](#outputs)
+  - [environment variables](#environment-variables)
+- [Keep up-to-date with GitHub Dependabot](#keep-up-to-date-with-github-dependabot)
+- [How can I help?](#how-can-i-help)
+- [License](#license)
```

Comment:
I prefer to keep `* ` for unordered lists. Please undo these changes.

Reply:
Sorry it was my toc plugin. Undoing them now...

Reply:
How is that?